libsrc/ImageSEGConverter.cpp

```diff
@@ -167,7 +168,8 @@ namespace dcmqi {
 
         LabelStatisticsType::BoundingBoxType bbox = labelStats->GetBoundingBox(label);
         unsigned firstSlice, lastSlice;
-        bool skipEmptySlices = true; // TODO: what to do with that line?
```

Comment:
what is the reason you commented it out? I think it would be better to test and check this functionality works, and use the flag passed to the function.